.github/workflows/build.yml

```diff
@@ -191,9 +191,9 @@ jobs:
         if: ${{ github.event_name == 'schedule' }}
         uses: azure/login@v1
         with:
-          client-id: ${{ secrets.AZURE_CLIENT_ID }}
-          tenant-id: ${{ secrets.AZURE_TENANT_ID }}
-          subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+          client-id: ${{ vars.AZURE_CLIENT_ID }}
+          tenant-id: ${{ vars.AZURE_TENANT_ID }}
+          subscription-id: ${{ vars.AZURE_SUBSCRIPTION_ID }}
```

Comment:
Hm, I thought these are sensitive.

Reply:
The sensitive one would have been `AZURE_CLIENT_SECRET` if any, but we're using OpenID Connect instead (https://github.com/iterative/dvc-bench/pull/456 & [original thread](https://iterativeai.slack.com/archives/C01R00PPQ1L/p1683613060582529)).

Reply:
Although those aren't any more sensitive than e.g. AWS account numbers, most people save the values together with `AZURE_CLIENT_SECRET` as secrets. 😅